media/plg_system_tour/js/tours.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    $(document).ready(function() {
```

Comment:
Let's not sacrifice quality here for the sake of making a statement about jQuery.  If there is no suitable vanilla library, then this is fine.  Software that is being abandoned, or has a questionable support state (shepherd support right now is questionable I'd say), isn't an option.